test/e2e/cluster_upgrade_test.go

```diff
@@ -91,20 +91,20 @@ var _ = Describe("When upgrading a workload cluster with a HA control plane", fu
 	})
 })
 
-var _ = Describe("When upgrading a workload cluster with a HA control plane using scale-in rollout", func() {
-	capi_e2e.ClusterUpgradeConformanceSpec(ctx, func() capi_e2e.ClusterUpgradeConformanceSpecInput {
-		return capi_e2e.ClusterUpgradeConformanceSpecInput{
-			E2EConfig:             e2eConfig,
-			ClusterctlConfigPath:  clusterctlConfigPath,
-			BootstrapClusterProxy: bootstrapClusterProxy,
-			ArtifactFolder:        artifactFolder,
-			SkipCleanup:           skipCleanup,
-			// This test is run in CI in parallel with other tests. To keep the test duration reasonable
-			// the conformance tests are skipped.
-			SkipConformanceTests:     true,
-			ControlPlaneMachineCount: pointer.Int64(3),
-			WorkerMachineCount:       pointer.Int64(1),
-			Flavor:                   pointer.String("kcp-scale-in"),
-		}
-	})
-})
+// var _ = Describe("When upgrading a workload cluster with a HA control plane using scale-in rollout", Label("k8s-upgrade", "slow", "network"), func() {
+// 	capi_e2e.ClusterUpgradeConformanceSpec(ctx, func() capi_e2e.ClusterUpgradeConformanceSpecInput {
+// 		return capi_e2e.ClusterUpgradeConformanceSpecInput{
+// 			E2EConfig:             e2eConfig,
+// 			ClusterctlConfigPath:  clusterctlConfigPath,
+// 			BootstrapClusterProxy: bootstrapClusterProxy,
+// 			ArtifactFolder:        artifactFolder,
+// 			SkipCleanup:           skipCleanup,
+// 			// This test is run in CI in parallel with other tests. To keep the test duration reasonable
+// 			// the conformance tests are skipped.
+// 			SkipConformanceTests:     true,
+// 			ControlPlaneMachineCount: pointer.Int64(3),
+// 			WorkerMachineCount:       pointer.Int64(1),
+// 			Flavor:                   pointer.String("kcp-scale-in"),
+// 		}
+// 	})
+//})
```

Comment:
Consider deleting it altogether instead of commenting it out

Reply:
we want to enable it after we test or add more features like clusterclass and topology etc. these are placeholders so that we dont miss on them

Reply:
added TODO comment